scripts/service-comparison.ts

```diff
@@ -167,8 +167,14 @@ async function runComparisons(environments = allEnvironments): Promise<void> {
       const ummRecord = ummRecordsMap[harmonyConfig.umm_s];
       const validationMessages = performValidations(ummRecord, harmonyConfig);
       if (validationMessages.length > 0) {
-        exitCode = 1;
-        console.log(`Validation failures for ${harmonyConfig.name} and ${ummRecord.meta['concept-id']}:\n    - ${validationMessages.join('\n    - ')}`);
+        // TODO this is a temporary check until the UMM records for this service chain are updated
```

Comment:
We're allowing for a difference in `concatenate_by_default` since for API users they want that to be false, but in EDSC they want the default to be to have the box checked. This isn't temporary, but the other shapefile discrepancy is.

I'm thinking instead we should still error out for the shape discrepancy until UMM-S is updated, and we shouldn't do any kind of temporary disable for that, but we should have code in to make the check for concatenate_by_default to always print a warning and not treat as an error.

Reply:
My build won't pass if this script errors, right?

Reply:
The compare services check will be marked as failed, but that does not block merging in the PR. It stays marked as failed so we know there is a change to UMM-S records that needs to happen.

Reply:
Fixed.